client/src/components/Hero.module.css

```diff
@@ -9,8 +9,8 @@
     display: flex;
```

Comment:
This is already true for 'Row' components.

Reply:
Done.